swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
+from uuid import uuid4
+condition_rating = {0:'Bad', 1: 'Fine', 2: 'Not Bad', 3:'Good',4:'Very good',5:'New'}
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid4().int
+        else:
+            self.id = id
+        self.condition= condition
+
+    def get_category(self):
+        category = self.__class__.__name__
+        return category
```

Comment:
As you get more comfortable with Python, feel free to combine these into one line!

```suggestion
        return self.__class__.__name__
```